src/index.js

```diff
@@ -0,0 +1,16 @@
+const tetrispiece = [
+    { shape: [[1,1,1,1]], colour: 'cerulean blue'},
+    { shape: [[1,1], [1,1]], colour: 'honey yellow'},
+    { shape: [[1,1,1], [1,0,0]], colour: 'flurescent pink'},
+    { shape: [[1,1,1], [0,0,1]], colour: 'brick red'},
+    { shape: [[1,1,1], [0,1,0]], colour: 'mint green'},   
+    { shape: [[1,1,0], [0,1,1]], colour: 'bauge'},
+    { shape: [[0,1,1], [1,1,0]], colour: 'rose gold'}
```

Comment:
Perhaps to think about, is this the easiest format to use in order to put the shape into the grid?

Seems likely that the grid will be full of numbers, where 0 = empty. Not sure what the other numbers will be used for (1 = red tile, 2 = yellow tile etc). For what it's worth keeping meaningful names for the colours is probably a good thing. Nothing to change for now